package.json

```diff
@@ -2,13 +2,25 @@
   "name": "owncloud-design-system",
   "version": "4.3.0",
   "description": "ownCloud Design System is based on VueDesign Systems and is used to design ownCloud UI components",
+  "keywords": [
+    "vue design system",
+    "design system",
+    "vue.js",
+    "vue",
+    "design",
+    "system"
+  ],
+  "homepage": "https://owncloud.github.io/owncloud-design-system/",
```

Comment:
Could be owncloud.design for a nice URL :)

Reply:
done, thanks.